spec/meilisearch/index/documents_spec.rb

```diff
@@ -116,20 +117,20 @@
       title = 'Hamlet'
       new_doc = { objectId: id, title: title }
       response = index.add_documents(new_doc)
-      sleep(0.1)
+      index.wait_for_pending_update(response['updateId'])
       expect(response).to be_a(Hash)
       expect(response).to have_key('updateId')
       expect(index.documents.count).to eq(documents.count + 1)
       expect(index.document(id)['title']).to eq(title)
       index.delete_document(id)
-      sleep(0.1)
+      index.wait_for_pending_update(response['updateId'])
```

Comment:
I cannot suggest, but I think your second wait_for.. is using the update id of the response off `add_documents` and not `delete_document`.

Reply:
Oh you're right! I failed my copy/paste 😂  shame on me!!